nixos/modules/services/logging/awstats.nix

```diff
@@ -110,7 +110,7 @@ in
         {
           "mysite" = {
             domain = "example.com";
-            logFile = "/var/spool/nginx/logs/access.log";
+            logFile = "/var/log/nginx/access.log";
```

Comment:
Do we have an awstats NixOS VM test? Did you manually test it still works?

Reply:
Checked on test VM. Awstats correct workinfg with this changes.